### PR TITLE
Fix grammar in install.rst

### PR DESCRIPTION
**Pull request by RafeyIqbalRahman**
### Summary
Fixed 3 grammatical errors in install.rst

### Details and comments
 The fixes are as follows:

- Changed "If the packages installed correctly ..." to "If the packages **are** installed correctly ...".
- Changed the phrase "It is worth point at ..." to "It is worth pointing that ...". Now it makes more sense.
- Added a comma to respect punctuation in a long, complex sentence.

### CODE REVIEW

**Review by SooluThomas — DISMISSED** [review]
LGTM! Thank you

**Comment by 3yakuya** [comment]
Very minor note, but I actually think "If packages installed (...)" is somehow more accurate, as it reflects the fact that they were supposed to be installed _just before _ user reads this bit.

Alternative change would be "got installed", though referring to software as "installing itself" is also somehow accepted (we often say "system is updating" or "packages are installing".)

**Comment by RafeyIqbalRahman** [comment]
> Very minor note, but I actually think "If packages installed (...)" is somehow more accurate, as it reflects the fact that they were supposed to be installed _just before _ user reads this bit.
> 
> Alternative change would be "got installed", though referring to software as "installing itself" is also somehow accepted (we often say "system is updating" or "packages are installing".)

I went with "got installed".

File: docs/install.rst [hunk 1]
```diff
@@ -82,8 +82,8 @@ recommended to install qiskit with the visualization extra requirements
 
   pip install qiskit[visualization]
 
-It is worth point at if you're a zsh user (which is the default shell on newer
-versions of macOS) you'll need to put ``qiskit[visualization]`` in quotes:
+It is worth pointing that if you're a zsh user (which is the default shell on newer
```
Comment (mtreinish) [inline docs/install.rst]:
```suggestion
It is worth pointing out that if you're a zsh user (which is the default shell on newer
```

File: docs/install.rst [hunk 2]
```diff
@@ -72,7 +72,7 @@ Next, install the Qiskit package, which includes Terra, Aer, Ignis, and Aqua.
   command will attempt to install qiskit-aer from sdist (source distribution)
   which will try to compile aer locally under the covers.
 
-If the packages installed correctly, you can run ``conda list`` to see the active
+If the packages got installed correctly, you can run ``conda list`` to see the active
```
Comment (mtreinish) [inline docs/install.rst]:
```suggestion
If the packages were installed correctly, you can run ``conda list`` to see the active
```

**Review by 3yakuya — APPROVED** [review]

**Review by mtreinish — APPROVED** [review]

**Pull request merged**